packages/node/rollup.config.js

```diff
@@ -40,6 +40,27 @@ const external = [
   })
 ];
 
+const onwarn = warning => {
+  // Silence circular dependency warnings specifically for reasonable
```

Comment:
What is a reasonable circular dependency?

Reply:
Updated the documentation